app/src/main/res/layout/activity_user.xml

```diff
@@ -1,151 +1,139 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:theme="@style/AppTheme.NoActionBar"
+    android:orientation="vertical"
     tools:context=".user.myProfile.UsersProfileActivity">
 
     <LinearLayout
         android:layout_width="match_parent"
-        android:layout_height="83dp"
-        android:layout_alignParentStart="true"
-        android:layout_alignParentTop="true"
-        android:layout_marginStart="0dp"
-        android:layout_marginTop="0dp"
-        android:background="@color/colorPrimary"
+        android:layout_height="wrap_content"
+        android:gravity="center_vertical"
+        android:background="@drawable/half_background"
         android:orientation="horizontal">
 
+        <RelativeLayout
+            android:layout_width="120dp"
+            android:layout_height="120dp"
+            android:layout_marginStart="@dimen/classic_margin">
+
+            <ImageView
+                android:id="@+id/profile_picture"
+                android:layout_width="120dp"
+                android:layout_height="120dp"
+                android:src="@drawable/unknown_person" />
+
+            <TextView
+                android:id="@+id/current_streak"
+                android:background="@drawable/ic_signal_cellular_4_bar_black_24dp"
+                android:layout_width="60dp"
+                android:layout_height="wrap_content"
+                android:layout_alignParentTop="true"
+                android:layout_alignParentEnd="true"
+                android:drawableEnd="@drawable/ic_flame"
+                android:paddingBottom="30dp"
+                android:textAlignment="viewEnd"
+                android:textColor="@android:color/white"
+                android:textSize="18sp" />
+
+        </RelativeLayout>
+
+        <TextView
+            android:id="@+id/user_name"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
+            android:text="USER"
+            android:textAlignment="center"
+            android:textSize="24sp"
+            android:textStyle="bold" />
     </LinearLayout>
 
-    <de.hdodenhof.circleimageview.CircleImageView xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
If we remove the circle images, we should remove the dependencies in the app.build as well